eland/operations.py

```diff
@@ -505,7 +541,9 @@ def _unpack_metric_aggs(
                     ]["value"]
 
                 # Null usually means there were no results.
-                if agg_value is None or np.isnan(agg_value):
+                if not isinstance(agg_value, (list, dict)) and (
```

Comment:
When is `agg_value` a list/dict? I figured agg_value is always a scalar.

Reply:
Yes, so lets see the result we fetch
|pd_agg|es_agg|agg_value|
|--------|--------|----------|
|quantile|('percentiles', [20.0, 50.0])|{'20.0': 361.0407676152134, '50.0': 640.3872852064159}|

after using the `agg_value` at line 508 (in my branch), 
Usually no point in having `agg_value` after this, hence to skip the comparisons below . I have to add that check.
also `np.isnan(agg_value)` fails if `agg_value` is `dict`.
E.g.
```python
>>> np.isnan({})      
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: ufunc 'isnan' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''
```
I am checking if agg_value will ever become `list`, If not ill remove it.